File: PRACTICA DE CAMPO 03-S07/C#/PracticaCampo_03_S07/s07_pc03_separador_pantalla.cs
int aumento = 0;

Console.WriteLine("Ingrese caracter :");
char texto_caracter = char.Parse(Console.ReadLine());
Console.WriteLine("Ingrese numero para repetir :");
int numero_veces = int.Parse(Console.ReadLine());

//string resultado = mostrar_separador(texto_caracter,numero_veces);
var resultado = mostrar_separador(texto_caracter,numero_veces);
Console.WriteLine(resultado.Item1);
Console.WriteLine(resultado.Item2);

(string,string) mostrar_separador(char _text_caracter, int _numero_veces){    
    string texto_separador = "";
    string texto_separador_02 = "";
    while (aumento < _numero_veces){        
        texto_separador = texto_separador + _text_caracter;
        aumento = aumento + 1;
    }
    for (int a=1;a<=_numero_veces;a++){
        texto_separador_02 = texto_separador_02 + _text_caracter;
    }
    
    return (texto_separador, texto_separador_02);
}

File: PRACTICA DE CAMPO 03-S07/PYTHON/s07_pc03_separador_pantalla.py
texto_caracter = input("Ingrese caracter :")
numero_veces = int(input("Ingrese numero para repetir :"))

def mostrar_separador(_texto_caracter,_numero_veces):
    texto_separador = ""
    texto_separador_02 = ""
    aumento = 0
    while aumento < _numero_veces:
        texto_separador = texto_separador + _texto_caracter
        aumento += 1

    for a in range(_numero_veces):
        texto_separador_02 = texto_separador_02 + _texto_caracter

    return texto_separador, texto_separador_02

separador1, separador2 = mostrar_separador(texto_caracter, numero_veces)
print(separador1)
print(separador2)

